src/workers/parser/getSymbols.js

```diff
@@ -265,10 +272,20 @@ function extractSymbol(path: SimplePath, symbols) {
       location: { start, end }
     });
   }
+
+  if (t.isAwaitExpression(path) || t.isYieldExpression(path)) {
+    const { start, end } = path.node.loc;
```

Comment:
I think we could try something like this for the left/right sibling.

So for this example: [ast](https://astexplorer.net/#/gist/cb2b761d20387a311a1d7187ce18c694/fec1187db78c84128d299cd816262c6255a6997c)
```js
async function foo(x, y) {
  console.log()
  await slower();
  return bar;
}
```

We can get `console.log` and `bar` locations by:

1. checking if the parent is an expression statement
2. using `simple-path.js` getSibling, to get the previous and next sibling.

```js
if (t.isExpressionStatement(path.parent)) {
  const prevNode = path.getSibling(-1);
  const prev = prevNode && t.isExpressionStatement(prevNode) ? prevNode.loc : null;
  const nextNode = path.getSibling(1);
  const next = nextNode && t.isExpressionStatement(nextNode) ? nextNode.loc : null;
}


We will want guards to make sure that we're:

1. handling cases where await is the first or last statement
2. we're not getting locations when await is not a child of a statement e.g `foo(1, await slow(), 2)`